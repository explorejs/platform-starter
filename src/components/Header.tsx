import Link from "next/link";
import React from "react";

function Header() {
  return (
    <nav>
      Platform Starter
      <Link href="/login">Login</Link>
    </nav>
  );
}

export default Header;
