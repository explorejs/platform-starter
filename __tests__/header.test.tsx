import { render } from "@testing-library/react";
import Header from "../src/components/Header";

describe("Header", () => {
  it("Looks Great", () => {
    const { asFragment } = render(<Header />);

    expect(asFragment()).toMatchSnapshot();
  });
});
