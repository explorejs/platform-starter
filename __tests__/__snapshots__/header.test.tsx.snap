// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Looks Great 1`] = `
<DocumentFragment>
  <nav>
    Platform Starter
    <a
      href="/login"
    >
      Login
    </a>
  </nav>
</DocumentFragment>
`;
